package.json

```diff
@@ -17,13 +17,13 @@
     "@matterlabs/prettier-config": "^1.0.3",
     "@typescript-eslint/eslint-plugin": "^6.7.4",
     "@typescript-eslint/parser": "^6.7.4",
+    "eslint": "^8.51.0",
```

Comment:
why move import order? we should not introduce any changes when not needed

Reply:
It was done automatically with `yarn` - I guess the convention changed. I can revert it back though